src/index.jsx

```diff
@@ -5,9 +5,7 @@ import './App.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-	<React.StrictMode>
-		<div className='wrapper'>
-			<App />
-		</div>
-	</React.StrictMode>
+	<div className='wrapper'>
+		<App />
+	</div>
```

Comment:
I'll admit that this is not the proper way to stop the double rendering. You could use a memoized run counter but the Jack Herrington videos probably have more appropriate ways to deal with it.